Makefile

```diff
@@ -108,9 +108,8 @@ reload: default
 	$(OR_EXEC) -p $$PWD/  -c $$PWD/conf/nginx.conf -s reload
 
 
-### install:          Install the apisix
```

Comment:
this line is useful, we can not delete it.

![image](https://user-images.githubusercontent.com/6814606/82825642-2e018000-9ede-11ea-8539-a27b1960a5d5.png)